webapp/tests/karma/ts/modules/messages/messages.component.spec.ts

```diff
@@ -12,6 +12,7 @@ import { MessageContactService } from '@mm-services/message-contact.service';
 import { RelativeDatePipe } from '@mm-pipes/date.pipe';
 import { SettingsService } from '@mm-services/settings.service';
 import { ModalService } from '@mm-modals/mm-modal/mm-modal';
+import { NavigationComponent } from '@mm-components/navigation/navigation.component';
```

Comment:
This import isn't used yet. But I should probably add this dependency to avoid warnings in the unit test 😅 I'll check this out in messages and also in analytics

Reply:
I added it just to avoid warnings in unit tests and keep the log cleaner.